01_Core/week04/ecommerce/script.js

```diff
@@ -0,0 +1,68 @@
+var slidesArray = [];
+var slideNumber = 0;
+
+var mainSlide0 = document.querySelector(".slide-1");
+var mainSlide1 = document.querySelector(".slide-2");
+var mainSlide2 = document.querySelector(".slide-3");
+slidesArray[0] = mainSlide0;
+slidesArray[1] = mainSlide1;
+slidesArray[2] = mainSlide2;
+
+
+
+/************************************Function to move slide forward*********************************/ 
+function slideNext () 
+{
+    console.log(slidesArray);
+    
+    if (slideNumber == 0)
+        {
+            slideNumber = slideNumber + 1;
+            mainSlide0.classList.toggle("show");
+            mainSlide1.classList.toggle("show");
+            console.log(slidesArray);
+            
+        }
+    else if (slideNumber == 1)
+        {
+            slideNumber = slideNumber + 1;
+            mainSlide1.classList.toggle("show");
+            mainSlide2.classList.toggle("show");
+        }
+    
+    else if (slideNumber == 2)
+        {
+            slideNumber = 0;
+            mainSlide2.classList.toggle("show");
+            mainSlide0.classList.toggle("show");
+        }
+    
+    
+}
+
```

Comment:
might be a way to tighten this up by using the slidenumber as the variable to toggle as well so you didnt need so many conditionals.